ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -124,8 +107,8 @@ vlan_interfaces:
     eos_cli: |
       {{ svi_raw_eos_cli | indent(6,false) }}
 {%             endif %}
-{%             if svi_structured_config is arista.avd.defined %}
-    struct_cfg: {{ svi_structured_config }}
+{%             if svi.structured_config is arista.avd.defined %}
+    struct_cfg: {{ svi.structured_config }}
```

Comment:
Is this working? it looks like we do something specific for structured config line 6 in this file and that we cannot directly access it via `svi.structured_config` - if this works like this then you could I guess remove lines 6 to 12 as well as we are not using the variable anymore.

Reply:
Ah, this was the cause of failed molecule test case - fixed, thank you @gmuloc

Reply:
fixed with ff96882